src/middlewares/authorization.middleware.ts

```diff
@@ -0,0 +1,52 @@
+import { NextFunction, Request, Response } from "express";
+import jwt from "jsonwebtoken";
+
+export const verifyUser = async (
+  req: Request,
+  res: Response,
+  next: NextFunction
+) => {
+  try {
+    if ((await req.cookies.session) || req.headers.authorization) {
```

Comment:
good. but do this seems like you can separate into a function call `getAccessToken(req: Request)`